bin/opencollective.js

```diff
@@ -0,0 +1,47 @@
+const chalk = require("chalk");
+const fs = require("fs");
+const path = require("path");
+
+// Only show emoji on OSx (Windows shell doesn't like them that much ¯\_(ツ)_/¯ )
+function emoji(emoji) {
+	if (process.stdout.isTTY && process.platform === "darwin") {
+		return emoji;
+	} else {
+		return "";
+	}
+}
+
+function print(str = "", color = "dim") {
+	const terminalCols = 80;
+	// eslint-disable-next-line no-control-regex
+	const ansiEscapeSeq = /\u001b\[[0-9]{1,2}m/g;
+	const strLength = str.replace(ansiEscapeSeq, "").length;
+	const leftPaddingLength = Math.floor((terminalCols - strLength) / 2);
+	const leftPadding = " ".repeat(leftPaddingLength);
+	str = chalk[color](str);
+	console.log(leftPadding, str);
+}
+
+function printBadge() {
+	console.log("\n");
+	print(`${chalk.bold("Thanks for using")} ${chalk.bold.blue("Webpack!")}`);
+	print(`Please consider donating to our ${chalk.bold.blue("Open Collective")}`);
+	print("to help us maintain this package.");
+	console.log("\n\n");
+	print(`${emoji("👉")} ${chalk.bold.yellow(" Donate:")} ${chalk.reset.underline.yellow("https://opencollective.com/webpack/donate")}`);
+	console.log("\n");
+}
+
+
+const now = new Date();
```

Comment:
just printing on mondays regardless of diff is ok. Could you remove the file with the timestamp? It's unneeded and will bloat our code.

Reply:
If we only check the day of the week, we will print every time the cli is executed on Monday.
I don't like the file too, other solutions are welcome

Reply:
https://nodejs.org/api/fs.html#fs_stats_atime

Reply:
I've updated the code using `atime`, thank you for the suggestion. Works on MacOS, I need to test it on Windows.
I'm busy with some exams, I'll keep working on this in the next days

Reply:
done